### DISCUSSION TITLE
Is there any example that facilitates simulation of sudden death due to Li plating?

### DISCUSSION THREAD

**Discussion by WoohyunJeong95** [pybamm-team/PyBaMM#4321] [Q&A]
I wanted to reproduce degradation patterns in Yang 2017 (Yang et al., Journal of Power Sources 360 (2017) 28-40), but failed. 
I have some battery cycling data and it shows a transition from linear to nonlinear aging (somewhat looks like the figure below).

![image](https://github.com/user-attachments/assets/3716aab4-ab12-4401-ad4b-04509941b092)
Is there any example that facilitates simulation of sudden death due to Li plating? 

I modified various parameters related to SEI growth and Li plating, but I couldn't generate the right curve.
I would appreciate any help.

**Comment by valentinsulzer** [comment]
I tried hard to replicate this paper a few years ago and couldn't. The model should be identical if you use the "ec reaction" SEI model, "irreversible" lithium plating model, and set "lithium plating porosity change" and "SEI porosity change" to "true". The problem is they don't share all the parameters of the model, since some of them are proprietary to GT Autolion. If you contact the authors they might send them to you privately.

**Reply by Thiyaga2025** [reply]
hi @WoohyunJeong95 ,

any progress with Yang's paper model. which parameter set you are using as base.
thanks.
